examples/DataApi-Water-And-Energy-Usage/template/modules/l0/src/main/scala/com/my/currency/l0/Main.scala

```diff
@@ -23,29 +23,32 @@ object Main
     ClusterId(UUID.fromString("517c3a05-9219-471b-a54c-21b7d72f4ae5")),
     version = BuildInfo.version
   ) {
-  def dataApplication:  Option[BaseDataApplicationL0Service[IO]] = Option(BaseDataApplicationL0Service(new DataApplicationL0Service[IO, Usage, State] {
-    override def genesis: State = State(0,0)
+  def dataApplication: Option[BaseDataApplicationL0Service[IO]] =
+    Option(BaseDataApplicationL0Service(new DataApplicationL0Service[IO, Update, State] {
 
-    override def validateData(oldState: State, updates: NonEmptyList[Signed[Usage]])(implicit context: L0NodeContext[IO]): IO[DataApplicationValidationErrorOr[Unit]] = Data.validateData(oldState, updates)
+    override def genesis: State = State(Map.empty)
```

Comment:
Can we call this out with a comment? `State` is a pretty important thing to understand if trying to extend this example.